prisma/datamodel.graphql

```diff
@@ -43,3 +45,39 @@ type Organization {
   acronymFr: String!
   teams: [Team!]! @relation(name:"Teams", onDelete: CASCADE) 
 }
+
+type Approval {
+    id: ID! @unique
+    gcIDApprover: Profile! @relation(name:"Approver")
+    gcIDSubmitter: Profile @relation(name:"Submitter")
+    requestedChange: RequestedChange!
+    createdOn: String!
+    actionedOn: String
+    deniedComment: String
+    status: Status!
+    changeType: ChangeType!
+}
+
+type RequestedChange {
+    id: ID! @unique
```

Comment:
Needed to add a RequestedChange id field becuase gcID is not a unique field to RequestedChange.